docs/source/index.rst

```diff
@@ -99,6 +67,17 @@ CellRank was developed in collaboration between the `Theislab`_ and the `Peerlab
     tutorials
     examples
 
+.. toctree::
```

Comment:
@Marius1311 I've modified it like this, please check

Reply:
I like it, thanks for the suggestion.